apps/website/src/components/show-and-tell/ShowAndTellEntryForm.tsx

```diff
@@ -578,6 +609,24 @@ export function ShowAndTellEntryForm({
           )}
         </Button>
       </div>
+
+      <ModalDialog
+        isOpen={!!previewImageUrl}
+        closeModal={closeModal}
+        title="Image Preview"
```

Comment:
```suggestion
        title="Image Preview"
        panelClassName="max-w-3xl"
```
I think we can make this a bit easier so the preview is more helpful